webview-ui/src/ClusterProperties/ClusterDisplay.tsx

```diff
@@ -62,6 +62,9 @@ export function ClusterDisplay(props: ClusterDisplayProps) {
     const showReconcileButton =
         props.clusterInfo.provisioningState === "Canceled" && props.clusterInfo.powerStateCode === "Running";
 
+    const supportedPatchVersions = props.clusterInfo.supportedVersions.flatMap((v) => v.patchVersions);
```

Comment:
@sprab, @Tatsinnit: I changed this logic slightly - can you please check it? Previously it was checking whether the version was supported by seeing whether the cluster version was a substring of one of the supported minor versions. This is now a stricter check, which looks at whether the cluster version matches one of the supported patch versions. Is this right?

Reply:
💡 Thanks, we just need to think of this might need @sprab , Pete and I might need some ideas/clarifications for this 

![image](https://github.com/Azure/vscode-aks-tools/assets/6233295/8f2d3d3c-0735-4520-99b7-25d6de698b73)

Reply:
@sprab: A specific question that might help us answer this: what does it mean for the user if the patch version is not in the list of supported versions, but the minor version is?

I agree with Tats that "out of support" is misleading in this case, because the minor version is still in support. But is there something "wrong" with the cluster if the patch version is not in the list?

Reply:
So, th fix for this fixed  the above 🥷 https://github.com/Azure/vscode-aks-tools/pull/538#discussion_r1516995823


<img width="1039" alt="Screenshot 2024-03-08 at 3 04 21 PM" src="https://github.com/Azure/vscode-aks-tools/assets/6233295/e9dc6e2d-f820-412a-8f77-ee7bcf2678fb">